### DISCUSSION TITLE
Making Image Occlusion cards

### DISCUSSION THREAD

**Discussion by debanjandhar12** [debanjandhar12/logseq-anki-sync#125] [Documentation]
Documentation has been moved from this location to this page: https://debanjandhar12.github.io/logseq-anki-sync/docs/card-types/image-occlusion

**Comment by busera** [comment]
Would it be possible to hide the occlusion block property?

**Reply by debanjandhar12** [reply]
Please read the discussion at https://github.com/debanjandhar12/logseq-anki-sync/issues/139. 
It detail some methods to hide the property.

**Comment by HerkulesX** [comment]
Is there a possibility for me to only uncover only my questioned (not all) occlusion(s)?

**Comment by ieanbqpreo572024** [comment]
Is it possible to automatically increment the `cloze id` as I go?

It is a pain to manually increase the `cloze ID` count every time I add a new box. It would be better to have a button:

[yes/no] increment the `ID` after each creation of a new box

**Comment by ieanbqpreo572024** [comment]
Further, I feel that you can enable this feature:

Now: click the `new box` button, then drag out a box. Click on a box, then the `remove box` button.

Suggested:
- Just drag on the picture to create a new box and _skip the `click the "new box" button` part_
-  Click on a box, then press `backspace` to remove the box

Thanks so much!

**Comment by LK0624** [comment]
if selected from google image drag drop, 

it does not work I think.

it has to be saved image? 
![CleanShot 2024-09-08 at 18 51 00](https://github.com/user-attachments/assets/fc0ccf54-f904-49f4-aebb-07fb08d6131f)
